AmplifyPlugins/Core/AWSPluginsCoreTests/Model/GraphQLDocument/GraphQLCreateMutationTests.swift

```diff
@@ -235,4 +235,42 @@ class GraphQLCreateMutationTests: XCTestCase {
         }
         XCTAssertEqual(input["commentPostId"] as? String, post.id)
     }
+
```

Comment:
General test note: make sure we're covering all permutations of schemas, including mutability and required.